### DISCUSSION TITLE
Loading or Pushing Multiplatform OCI Image Tarball

### DISCUSSION THREAD

**Discussion by tunahanertekin** [docker/buildx#1585] [Q&A]
Hey there!

I have a tarball image which is built with `docker buildx build` for platforms `linux/amd64` and `linux/arm64` with argument `--output=type=oci,dest=/some/path`. I am not able to go back and rebuild image with `--load` or `--push`. 

- My first question is, how can I push this tarball to an image registry?
- And secondly, how can I instantly use this image in my machine which is `linux/amd64`?

I would be appreciated if anyone can help me on this.

ps. I have tried to import the tarball image using `docker import` but I got the error below everytime I tried:
```
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "submariner-operator": executable file not found in $PATH: unknown.
ERRO[0000] error waiting for container: context canceled
```

And `docker load` gives this error:
```
open /var/lib/docker/tmp/docker-import-1168996138/blobs/json: no such file or directory
```

**Comment by sansmoraxz** [comment]
Did you figure it out?
